### PR TITLE
feat: Add images and inputs components

### PR DESCRIPTION
**Pull request by rahulbakshi4**
Features Added:
1. Images
	- Square Images
	- Round Images
	- Responsive Image
2. Inputs
	- Primary Input
	- Error Input
	- Disabled Input

### CODE REVIEW

**Comment by SJTGSHIVAM** [comment]
Changes look good I believe we are good to merge it!

File: Components/Inputs/input.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+                <label for="username">Username</label>
+                <input type="text" class="inputs input-primary" />
+
+                <label for="password">Password</label>
+                <input type="password" class="inputs input-error" />
+                <span class="error-text">Incorrect Password</span>
+
+                <label for="email">Email</label>
+                <input type="text" class="inputs input-disabled" disabled />
```
Comment (mansipadgelwar) [inline Components/Inputs/input.html:87]: Here you can add type="email"

**Review by mansipadgelwar — APPROVED** [review]

File: Components/Inputs/input.html (hunk 1 above)
Comment (rahulbakshi4) [reply Components/Inputs/input.html:87]: Thanks for suggestion adding it

**Review by anuj-kumary — COMMENTED** [review]
I check your input components, you have to add textarea components.

**Pull request merged**